projects/dashboards/gfbio/src/app/app.component.ts

```diff
@@ -79,6 +78,8 @@ export class AppComponent implements OnInit, AfterViewInit {
 
     private windowHeight$ = new BehaviorSubject<number>(window.innerHeight);
 
+    private GFBIO_COLLECTIONS_DATA_PROVIDER_ID = 'f64e2d5b-3b80-476a-83f5-c330956b2909';
```

Comment:
I guess it is not a task for now, but we should think about configuring our GIS.

Reply:
Once we have stable names for stuff, we should use the stable name of the GFBio Collections provider here instead of a cryptic UUID. IMHO this here is nothing that needs configuration.